bugbug/model.py

```diff
@@ -210,6 +210,12 @@ def classify(
 
         return classes
 
+    def check(self):
```

Comment:
I'm thinking that in the end we will have two kinds of checks.
One kind of check to perform after training on the model itself (e.g. the check on the accuracy), another kind of check to be done periodically to make sure the data on which we trained on is still good (e.g. the check you are implementing here).

The first kind of check will be executed after training.
The second kind of check will be executed periodically (and maybe we'll have to retrigger the pipeline in case of errors during the check, to retrain).

So we're likely going to have two functions in the end (but for now just one function is OK, as we are only adding the second kind at the moment).

Reply:
I see, maybe the first check could be part of the training itself, does it makes sense to save a model with a bad accuracy?

Reply:
Right, we could fail the training task when the accuracy is too bad